packages/monaco/src/browser/monaco-editor-model.ts

```diff
@@ -329,6 +329,7 @@ export class MonacoEditorModel implements ITextEditorModel, TextEditorDocument {
                 await operation();
             } catch (e) {
                 console.error(e);
+                throw e;
```

Comment:
it does not throw on purpose to avoid rejecting consequent operations

Reply:
Thanks for the note. Well, we should reject it somehow. That would be the desired flow when clients want to handle errors. If the error was swallowed on purpose, that's a problem. What was the problematic use-case?

Reply:
I can filter the error and rethrow only when it indicates t a connection disposed issue, but the rejection must happen, preferably always.

Reply:
maybe something like:
```
const result = this.pendingOperation.then(() = operation());
this.pendingOperation = result.catch(e => console.error(e));
return result;
```

Reply:
OK, I will check the call sites. Thanks! Eventually, I am fine with introducing a new template method, `handleError`, with `console.log(error)` default implementation in the `monaco-editor-model.ts` module, but best would be to have the error handling in Theia.